### ISSUE TITLE
Possible to set CT values instead of RGB?

### ISSUE THREAD

**Issue by jeanofthedead** [kpsuperplane/homebridge-wiz-lan#63]
Hello,

I was wondering if there was a way to manually set color temperature (CT) values for Philips WIz bulbs using HomeKit. Another developer on the Hubitat side of things has created a device code for these bulbs [here](https://github.com/zranger1/HubitatWizLightDriver), which allows users to set the CT values (in addition to RGB/HSV). This lets us control the brighter light settings as well as the pre-installed scenes (CT values #6000-6032). I was wondering if there was any way to control these capabilities in Homebridge, as well.

TL;DR - If I could get Siri to turn any of these bulbs a true _white_ (or activate any of their pre-installed scenes without Siri shortcuts) using my voice, I would be so happy.

Thanks!

**Comment by kpsuperplane** [comment]
Please see https://github.com/kpsuperplane/homebridge-wiz-lan#color. It's out of my hands unfortunately
